addons/account/models/account_move.py

```diff
@@ -3216,8 +3216,9 @@ def _notify_get_groups(self, msg_vals=None):
         self.ensure_one()
         if self.move_type != 'entry':
             for group_name, _group_method, group_data in groups:
-                if group_name == 'portal_customer':
+                if group_name in ('portal_customer', 'customer'):
                     group_data['has_button_access'] = True
+                    group_data['notification_is_customer'] = True
```

Comment:
We should add a comment to explain why this is needed here so in 1 year, when looking at it, we don't just think "what is this?" :grin:

Reply:
Done!